02/app.js

```diff
@@ -1,7 +1,28 @@
 
 /* rozwiązanie z pętlą for */
-const x = 4;
+const x = parseInt(prompt("podaj liczbę"));
 
+if(x>=1 && x<10){
+    for(let i=1;i<10;i++){
+     let result = x * i;
+    console.log( x,  "x",  i,  "=", result);
+}
```

Comment:
Uwaga na formatowanie, powinien być odstęp taki, aby był ten znak na tej samej wysokości co `for`